lib/testing/assertions/custom-matchers.test.ts

```
@@ -0,0 +1,269 @@
+        let value = 'initial'
+        
+        // Simulate async state change
+        setTimeout(() => {
+          value = 'final'
+        }, 100)
+        
+        await expect(() => value).toEventuallyEqual('final')
+      })
+
+      it('should timeout if value never matches', async () => {
+        let value = 'initial'
```

ESLint

[error] 'value' is never reassigned. Use 'const' instead.

(prefer-const)